tests/test_regrid.py

```diff
@@ -0,0 +1,131 @@
+"""Unit test for regridding."""
+
+from pathlib import Path
+import numpy as np
+import pytest
+from zampy.datasets.dataset_protocol import SpatialBounds
+from zampy.datasets.eth_canopy_height import parse_tiff_file
+from zampy.utils import regrid
+
+
+path_dummy_data = Path(__file__).resolve().parent / "test_data" / "eth-canopy-height"
+XESMF_INSTALLED = True
+try:
+    import xesmf as _  # noqa: F401 (unused import)
+except ImportError:
+    XESMF_INSTALLED = False
+
```

Comment:
Can you add a comment on why we need this code block?

Reply:
Good point. A comment is now added to the code:
```py
# Since xesmf is only supported via conda, we need these checks to support
# tests cases running with and without conda environment in CD/CI
```